radix-engine-interface/src/blueprints/resource/bucket.rs

```diff
@@ -60,8 +60,40 @@ pub const BUCKET_CREATE_PROOF_IDENT: &str = "Bucket_create_proof";
 #[derive(Debug, Clone, Eq, PartialEq, ScryptoSbor)]
 pub struct BucketCreateProofInput {}
 
+#[derive(Debug, Clone, Eq, PartialEq, ScryptoSbor)]
+pub struct BucketLockAmountInput {
```

Comment:
Do we really want BucketLockAmount? I think conceptually easier if we just don't allow this. For example, this can be done at the application layer by `bucket.take().lock()`.